package.json

```diff
@@ -187,6 +203,7 @@
     "unified": "^9.0.0",
     "unist-util-find": "^1.0.1",
     "unist-util-visit": "^2.0.2",
+    "@thomaskoppelaar/markdown-it-wikilinks": "1.3.0",
```

Comment:
Should this be a separate repo, or should some of this code be moved in as well?

Reply:
I believe it's best to keep this separate - We can override practically all settings / functions from within `require('@thomaskoppelaar/markdown-it-wikilinks')({})`, and people may want to use this themselves for exporting / rendering for their own personal use cases.